packages/block-editor/src/components/inner-blocks/index.js

```diff
@@ -24,13 +24,11 @@ import DefaultBlockAppender from './default-block-appender';
  */
 import BlockList from '../block-list';
 import { withBlockEditContext } from '../block-edit/context';
+import TemplatePicker from './template-picker';
 
 class InnerBlocks extends Component {
 	constructor() {
 		super( ...arguments );
-		this.state = {
-			templateInProcess: !! this.props.template,
-		};
```

Comment:
The e2e tests show a failure because of this. I think it ensures the InnerBlocks don't focus the placeholder to avoid creating any extra block while the template is being processed (or something like that).

@jorgefilipecosta might now more, I think it would be good to document.

Right now, I'm going to restore it.